rq/worker.py

```diff
@@ -155,9 +155,11 @@ def find_by_key(cls, worker_key, connection=None, job_class=None,
 
         return worker
 
-    def __init__(self, queues, name=None, default_result_ttl=None, connection=None,
-                 exc_handler=None, exception_handlers=None, default_worker_ttl=None,
-                 job_class=None, queue_class=None, default_job_monitoring_interval=None):  # noqa
+    def __init__(self, queues, name=None, default_result_ttl=DEFAULT_RESULT_TTL,
+                 connection=None, exc_handler=None, exception_handlers=None,
+                 default_worker_ttl=DEFAULT_WORKER_TTL, job_class=None,
+                 queue_class=None,
+                 default_job_monitoring_interval=DEFAULT_JOB_MONITORING_INTERVAL):  # noqa
```

Comment:
Could you please change the keyword `default_job_monitoring_interval` to `job_monitoring_interval`?

Reply:
Done.